docker/readme.md

```diff
@@ -149,3 +154,21 @@ RAVEN_ARGS='--log-to-console'
 ##### How to set a custom config file?
 
 Mount it as a docker volume and use `--config-path PATH_TO_CONFIG` command line argument in order to use settings file from outside of server directory.
+
+##### What migration process from 5.4 image looks like?
```

Comment:
where is the link to actually see this in the webstie?

Reply:
PR is still opened: https://github.com/ravendb/docs/pull/1665